components/ai_chat/resources/page/state/conversation_context.tsx

```diff
@@ -439,6 +439,10 @@ export function ConversationContextProvider(props: React.PropsWithChildren) {
       aiChatContext.dismissStorageNotice()
     }
 
+    if (aiChatContext.isStandalone) {
+      getAPI().metrics.willSendPromptWithFullPage()
```

Comment:
note that for android everthing is standalone / fullpage, even though the conversation might be tab-related